We do not need to check coverage for this project.
See also https://github.com/briancurtin/deprecation/issues/67

--- deprecation-2.1.0/test-requirements.txt.orig
+++ deprecation-2.1.0/test-requirements.txt
@@ -1,5 +1,3 @@
-codecov>=2.0
-coverage>=4.3
 flake8>=3.2
 packaging>=16.8
 unittest2>=1.1;python_version<'3.5'
